README.md

```diff
@@ -58,7 +58,10 @@ but are always run in the CI.
 
 To download the test files locally, run `./download_test_data.sh`.
 It will ask for username and password and requires `wget` to be installed.
-Ask one of the project maintainers for the credentials.
+Username is "lstchain". Ask one of the project maintainers for the password. If 
```

Comment:
what about including username and password both in the mentioned link?

Reply:
Done